### PR TITLE
Enhancement and bug fix

### PR DESCRIPTION
**Pull request by rainasong**
# Description

Fixed a list of bugs on frontend:
- Removed force refresh
- Refresh now stay on the same page
- Navbar now corresponding to the page
- Added loading for create event
- Fixed text overflow issues
- Fixed numerous styling issues

Fixes/resolves # (issue)

## Screenshots

Please include any screenshots or media that illustrates the changes made

## Type of change

Please delete options that are not relevant.

- [x] **Bug fix** (non-breaking change which fixes an issue)
- [ ] **New feature** (non-breaking change which adds functionality)
- [ ] **Improvement** (non-breaking change which improves existing functionality)
- [ ] **Breaking change** (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:

Leave blank if not applicable

I have completed these steps when making this pull request:

- [ ] I have assigned my name to the issue
- [ ] I have labelled the PR appropriately
- [ ] I have assigned myself to the PR

Before opening the PR for review:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: packages/frontend/pages/login.tsx [hunk 1]
```diff
@@ -16,10 +16,6 @@ const Login: NextPage = () => {
     useAuth();
   const router = useRouter();
 
-  const BASE_URL =
-    process.env.NEXT_PUBLIC_SOCKET_URL! +
-    (process.env.NEXT_PUBLIC_BASE_URL ?? 'api/v1');
-
```
Comment (wqsz7xn) [inline packages/frontend/pages/login.tsx:22]: Was this an unused var within the file?

**Review by Sparye — APPROVED** [review]
c65c1cbd59e614bbb5902ac78ddf2d62
LGTM!

**Review by wqsz7xn — APPROVED** [review]

File: packages/frontend/pages/login.tsx (hunk 1 above)
Comment (rainasong) [reply packages/frontend/pages/login.tsx:22]: yep

**Comment by wqsz7xn** [comment]
https://www.youtube.com/watch?v=DNeikvoqQ-s

**Pull request merged**